src/components/Subscriptions.js

```diff
@@ -7,6 +7,17 @@ import createBillingAgreement from '../helpers/createBillingAgreement'
 import queryString from 'query-string'
 import '../assets/Subscriptions.css'
 
+let membership = props => {
+  let info
+  if (queryString.parse(props.location.search).plan === 'premiummob') {
```

Comment:
you can have a variable on line 11, `const plan = queryString.parse(props.location.search).plan'